### ISSUE TITLE
Failed error trying to describe product codes

### ISSUE THREAD

**Issue by DarraghDuffy** [newsapps/beeswithmachineguns#232]
Hi,

I am running the following command:
./bees up -s 2 -g bees -k bees -z eu-west-1b -i ami-8b8f59f2

And get this error:

![image](https://user-images.githubusercontent.com/3587980/67164853-9e5b8800-f376-11e9-9bf8-ddb6fec3769c.png)

Any help would very much be appreciated.

**Comment by mimart0** [comment]
I'm having the same issue.  Did you ever figure it out?

**Comment by DarraghDuffy** [comment]
Yes, it was simply the AWS zone, I was using Ireland, instead of a zone in the US. 

kind Regards
Darragh.
